community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/cache/OffHeapByteArray.java

```diff
@@ -98,28 +98,54 @@ public byte getByte( long index, int offset )
     @Override
     public short getShort( long index, int offset )
     {
-        return UnsafeUtil.getShort( address( index, offset ) );
+        return putShort( address( index, offset ) );
+    }
+
+    private short putShort( long p )
```

Comment:
Yup, I knew I renamed the wrong method at some point, but couldn't find it right then and then forgot about it